### DISCUSSION TITLE
Does Flatcar support LXC ?

### DISCUSSION THREAD

**Discussion by mikucat0309** [flatcar/Flatcar#676] [Q&A] [answered]
In [storage site](https://stable.release.flatcar-linux.net/amd64-usr/current/), there is a file named [flatcar-container.tar.gz](https://stable.release.flatcar-linux.net/amd64-usr/current/flatcar-container.tar.gz)
I use that file to create a LXC instance in my Proxmox VE and it boot sucessfully
it seems a LXC template, but no docs talk about LXC, [interop-matrix](https://github.com/flatcar-linux/Flatcar/blob/main/interop-matrix.md) neither
So what is that file's purpose ? Any instrcution or source about it ?

**Comment by pothos** [comment, marked as answer]
It's generated and can be a base for a container as with LXC but it has no designated use case it is tailored for and isn't tested. If you have a nice setup to share, how about adding it to the docs?
We have a section for things that don't go through our automated release validation: https://www.flatcar.org/docs/latest/installing/community-platforms/

**Comment by daper** [comment]
Hi @mikucat0309 could you share a bit more info on how did you managed to make it run?

**Reply by AnderssonPeter** [reply]
I'm also interested in how you managed to get it running!

**Comment by till** [comment]
You can use the container tarball as a rootfs with `lxc-create` with something like:

```
lxc-create -t local -n flatcar -- --fstree flatcar-container.tar.gz --metadata meta.tar.gz
```

The meta tarball needs a file config in it. Can be empty. So give that a try.

I don't currently have an lxc host, so the above is copied from my notes from a while ago.

**Comment by tormath1** [comment]
FWIW a Flatcar user shared this recently in the Matrix channel: https://gist.github.com/joshenders/1f67ceacaadc066e43e2312119e03533

**Comment by Bush-cat** [comment]
I installed flatcar as LXC on Proxmox, had to do some changes to the rootfs because it couldn't boot. It errors couldn't find /sbin/init. It seemed like the template didn't have any symlinks
so I mounted the image and made them myself, incase someone else has the problem
```bash
ln -s usr/bin bin
ln -s usr/sbin sbin
ln -s usr/lib lib
ln -s usr/lib64 lib64
```

**Comment by Bush-cat** [comment]
Flatcar starts and all but I couldn't manage to get ignition working, I put my ignition file at `/opt/org.flatcar-linux/config` and ran `flatcar-reset` but nothing was applied
